### PR TITLE
Organization Role

### PR DESCRIPTION
**Pull request by hindman-redhat**
Draft pull request

### CODE REVIEW

**Comment by willtome** [comment]
@hindman-redhat, currently the variables data structure would look like this
```
foreman_organizations: 
  - name: raleigh
    label: rdu
    server_url: url
    username: user
    password: pass
    validate_certs: no
    state: present
  - name: lanai 
    description: pacific datacenter 
    server_url: url
    username: user
    password: pass
    validate_certs: no
    state: present
  - name: default
    label: boring
    server_url: url
    username: user
    password: pass
    validate_certs: no
    state: present
```

Update so that the authentication vars, `server_url`, `username`, `password`, etc, so they are given once per role invocation with a list of the unique variables `name`, `description`, `label`, etc given as `foreman_organization`. See example below:
```
foreman_server_url: url
foreman_username: user
foreman_password: pass
foreman_validate_certs: no
foreman_organizations: 
  - name: raleigh
    label: rdu
    state: present
  - name: lanai 
    description: pacific datacenter 
  - name: default
    label: boring
    state: absent
```

the first set of authentication vars will go under the "Variables" section of your README and the `foreman_organizations` structure gets documented under the "Organization Data Structure" section of your README.

- [x] update loop vars
- [x] update README
- [x] update examples

**Comment by willtome** [comment]
Closes #1109

File: roles/organizations/README.md
Comment (willtome) [inline roles/organizations/README.md]: update to fully qualified collection name `theforeman.foreman.organizations`

File: roles/organizations/README.md
Comment (willtome) [inline roles/organizations/README.md]: `present` is not a valid default for `foreman_server_url`

File: roles/organizations/README.md
Comment (willtome) [inline roles/organizations/README.md]: username and password should not be required. check code for `default(omit)` to verify

File: roles/organizations/README.md
Comment (willtome) [inline roles/organizations/README.md]: use `foreman_` prefix for validate_certs variable name. document default value

File: roles/organizations/README.md
Comment (willtome) [inline roles/organizations/README.md]: `foreman_description` doesn't make sense here. perhaps you meant `foreman_organizations` as shown in your example?

File: roles/organizations/README.md
Comment (willtome) [inline roles/organizations/README.md]: no `foreman_` prefix required

File: roles/organizations/README.md
Comment (willtome) [inline roles/organizations/README.md]: False is not valid default for label

**Review by willtome — CHANGES_REQUESTED** [review]

File: roles/organizations/README.md
Comment (willtome) [inline roles/organizations/README.md]: Default would be `[]`

File: roles/organizations/README.md
Comment (willtome) [inline roles/organizations/README.md]: Default would be `""`

**Review by willtome — CHANGES_REQUESTED** [review]

**Review by willtome — APPROVED** [review]

**Comment by ehelms** [comment]
This diverges from our current variable naming conventions and therefore I believe it should be blocked on either updating to our current standard or waiting until we resolve the proposed naming changes so that we have consistency across our roles. 

The issue under consideration: https://github.com/theforeman/foreman-ansible-modules/issues/1118

**Comment by willtome** [comment]
Opened #1131 to meet the current naming scheme. Leaving the PR as is until #1118 is resolved and this can be merged or closed.

**Comment by evgeni** [comment]
Given we ended up with having a prefix, this is the way to go :)

**Comment by evgeni** [comment]
taken over in #1204

**Pull request closed without merging**
